lsp_bridge.py

```diff
@@ -191,6 +191,11 @@ def _open_file(self, filepath):
                 return False
             
             lang_server_info = load_single_server_info(single_lang_server)
+            if "omnisharp" in single_lang_server and get_os_name() == "windows":
+                user_emacs_dir = get_emacs_func_result("get-user-emacs-directory")
+                full_path = str(pathlib.Path(user_emacs_dir, lang_server_info["command"][0]))
+                lang_server_info["command"][0] = full_path
+
```

Comment:
We replace `%USER_EMACS_DIRECTORY%` with get-user-emacs-directory here, not write `omnisharp` here, lsp-bridge should not hardcode lsp server name.